### ISSUE TITLE
Change fast forward behavior (or add setting to change it)

### ISSUE THREAD

**Issue by sudoshindo** [rebelonion/Dantotsu#493]
Right now, the fast forward feature **doubles the current playback speed**.

If the playback speed is 1.5 then fast forward will be 3.0 speed.

My suggestion is to make fast forward use a fixed 2.0 speed (basically the same as YouTube's player). I think 2x is enough for most cases and anything higher just becomes hard to follow.

Alternatively, a new setting under `Anime > Player > Behavior ` that can swap the fast forward implementation would also be great for users who prefer going beyond 2x speed.

**Comment by rebelonion** [comment]
It used to stay at 2x, but it was a deliberate change.

**Comment by sudoshindo** [comment]
Will you consider adding a setting?

**Comment by rebelonion** [comment]
Yes.
